app/wait-for-local-db.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# wait-for-local-db
+
+set -e
+
+# Color formatting
+RED='\033[0;31m'
+NO_COLOR='\033[0m'
+
+MAX_WAIT_TIME=30 # seconds
+wait_time=0
```

Comment:
nit: i have a preference to have all bash variables be upper case, so this would be `WAIT_TIME`

Reply:
It seems like there are a few conventions, some that prefer lowercase for script-local variables: https://google.github.io/styleguide/shellguide.html#s7-naming-conventions

But I also see a lot of arguments both ways online.

One benefit of lower case is not needing to put `$` in front of it everywhere which is useful since it's referenced a few times. Thoughts?

Reply:
ah ok, but maybe for consistency then we can do all uppercase? (since i didn't realize about the lower case convention when working on infra template bin scripts)